recce/dbt.py

```diff
@@ -498,6 +498,7 @@ def callback(x):
 
     def columns_value_mismatched_summary(self, primary_key: str, model: str):
         column_groups = {}
+        errors = self.verify_primary_key(primary_key, model)
```

Comment:
Please make the validation error failed fast. That is, if the PK is not unque and with null, don't do the following check. It will query wrong result and potentially have at most n^2 records to query. (e.g  in a 1,000 rows table, if we select a all null column, it will get 1,000,000 outer join result)

Reply:
got if, skip execution when find any errors